src/api/app/controllers/webui/staging/projects_controller.rb

```diff
@@ -52,6 +58,10 @@ def set_staging_workflow
         flash[:error] = "Staging with id = #{params[:staging_workflow_id]} doesn't exist"
         return
       end
+
+      def set_bootstrap_views
```

Comment:
[nitpick] shouldn't this be in the webui controller? I mean the implementation itself, not the before_action.

Reply:
And please rename this to `switch_to_webui2` so we will not forget this when we finishing the bootstrap story.

Reply:
@hennevogel  I can't do that because the original `switch_to_webui2` is already defined in the same controller. Instead of that, I have changed its name and added a comment in order to find it when we search in the future:

```
+  # NOTE: remove when bootstrap migration is done (related to switch_to_webui2)
+  def set_webui2_views
```

Reply:
Also good :-)